README.md

```diff
@@ -26,153 +26,166 @@
 
 ## Options
 
-- Config your excute tool by `g:dashboard_default_executive`,This option mean what fuzzy
-  search plugins that you used. [vim-clap](https://github.com/liuchengxu/vim-clap)
-  and [fzf.vim](https://github.com/junegunn/fzf.vim)
-  [Telescope](https://github.com/nvim-lua/telescope.nvim)
-  You must instll one of them.
-
-  ```viml
+- Use `g:dashboard_default_executive` to select which fuzzy search plugins that you would like to apply: 
+  
+  - [vim-clap](https://github.com/liuchengxu/vim-clap)
+  
+  - [fzf.vim](https://github.com/junegunn/fzf.vim)
+
+  - [Telescope](https://github.com/nvim-lua/telescope.nvim)
+  
+    **You must install one of them.**
+  
+  ```vim
   " Default value is clap
   let g:dashboard_default_executive ='clap'
   ```
+  
+- Dashboard utilises some `vim-clap` and `fzf.vim` commands and displays the result in popup windows.
```

Comment:
I think we also should be add telescope into here.

Reply:
Unfortunately, I have no experience with Telescope. If there are people using it who would like to help with the process, that would be awesome. Otherwise, I mean I can dig in its docs a little bit and finish this part maybe.

Reply:
you can find doc in here https://github.com/nvim-telescope/telescope.nvim